source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,256 @@
+.. _Rust: https://www.rust-lang.org/
+
+.. _piggy-bank-preparing:
+
+====================================
+Preparing a testnet node on a server
+====================================
+
+This is the third :ref:`part of a tutorial<piggy-bank>` on smart contract
+development.
+
+So far you have written and tested a piggy bank smart contract in the Rust_ programming
+language.
+
+This part will focus on how you can set up your own testnet node on a server and create a testnet account.
+In more detail, you will create an instance on a server that will host your own testnet node. Secondly, you will create your own testnet account and prepare it so you can use it in the fourth :ref:`part of this tutorial<piggy-bank-deploying>` for deploying and interacting with your piggy bank smart contract.
+
+.. warning::
+
+   The reader is assumed to have basic knowledge of what a blockchain and smart
+   contract is, and some experience with Rust_.
+
+
+Preparation
+===========
+
+Before you start, make sure you create an account on your favourite cloud provider to set up your instance.
```

Comment:
Consider adding something like: `.. unless you intend to run a testnet locally on your computer, see more below.`

Since we mention that it is possible in the note below.